CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [3.0.0-beta]
```

Comment:
Should we do `beta1` just in case?

Reply:
Would that make people expect more betas? If they see `beta1` they may think `beta2` is coming. Since we only plan on having one (unless I missed something) I would stay with `beta` for now

Reply:
That's true. I think it'd just be in case there are problems with the release that small changes could be numbered. `-beta.0` might be cleared for showing that, but we could also just number afterwards.

Reply:
I'm wrong -- sorry. Semver says

> Example: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
https://semver.org/